### DISCUSSION TITLE
Why are the IMAGENET_MEAN and IMAGENET_STD values so large?

### DISCUSSION THREAD

**Discussion by ryuhei1112** [libffcv/ffcv#361] [Q&A] [answered]
Generally it is 
IMAGENET_MEAN=[0.485, 0.456, 0.406],
IMAGENET_STD=[0.229, 0.224, 0.225]
, but in FFCV it is 
IMAGENET_MEAN = np.array([0.485, 0.456, 0.406]) * 255
IMAGENET_STD = np.array([0.229, 0.224, 0.225]) * 255
Is there a reason for this?
I apologize if I have overlooked something.

**Comment by andrewilyas** [comment, marked as answer]
In FFCV, normalization is done before the image is rescaled to be in [0, 1] - so the images are still integers between 0 and 255 (their raw pixel values), which is why the normalization is rescaled too!

**Reply by ryuhei1112** [reply]
Thank you for your reply.
Your reply helped me to understand.

One more thing I'm wondering, I know training with DDP is common in FFCV, but is the batch normalization layer done correctly?

Is the batch normalization layer processed correctly without explicitly using torch.nn.SyncBatchNorm?
https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html
